domain/src/main/java/org/sagebionetworks/research/domain/step/implementations/UIStepBase.java

```diff
@@ -64,6 +66,9 @@
     @Nullable
     private final String footnote;
 
+    @NonNull
+    private final ImmutableSet<String> hiddenActions;
```

Comment:
Is this managing the set of actions to hide in the UI? Are we doing something like what is done to get the default ActionViews where there is a hierarchy that goes Step,Task,Default for defining which actions get hidden?

Reply:
I added this to the model since I saw it was added to iOS. I'd imagine step/task in the presentation layer would use these so that action hiding can be completely JSON driven if you don't make custom steps